text/0002-rfc-process.md

```diff
@@ -0,0 +1,485 @@
+- Feature Name: `rfc_process`
+- Start Date: 2024-02-05
+- RFC PR: [lf-lang/rfcs#2](https://github.com/lf-lang/rfcs/pull/2)
+- Tracking Issue(s): None
+
+# Abstract
+[abstract]: #abstract
+
+Establish a Request for Comments (RFC) process. The RFC process is intended to
+provide a consistent and controlled path for proposing new features to the
+Lingua Franca language, the compiler infrastructure and tooling, as well as the
+runtime libraries. The goal is to allow all stakeholders to participate in the
+design of new features and to establish confidence in the direction the language
+and tooling is involving in.
+
+# Motivation
+[motivation]: #motivation
+
+Lingua Franca started as a research project and major contributions to the
+language and infrastructure have been mostly driven by the research goals of
+individual contributors. While this approach allowed for quickly building a rich
+set of features, it also created significant heterogeneity within the project.
+To further scale the development effort, to professionalize the code base, and
+to establish Lingua Franca as a mature platform, a more uniform and disciplined
+approach is required.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Text to be placed in the README of this repository:
+
+---
+
+[Lingua Franca RFCs]: #lf-rfcs
+
+The RFC (request for comments) process is a framework for managing both
+historical and planned changes within the LF community. The record of active
+RFCs tells the story of our project, our technology and our thought process. We
+believe in an honest and open dialog between all contributors and stakeholders
+within the community. The RFC process establishes accountability, helps to
+identify and navigate disparate goals, and builds confidence in the direction
+the language and the tooling are evolving in.
+
+Many changes, including bug fixes and documentation improvements, can be
+implemented and reviewed via the normal GitHub pull request workflow. However,
+some changes are more substantial and we ask to put them through a bit of a
+design process. In particular, changes that impact other contributors or
+stakeholders in the LF community or changes that considerably impact the user
+experience. See [When you need to follow this process] for more details.
+
+## TL;DR
+
+Follow these steps to open a new RFC:
+1. Fork or clone this repository.
+2. Copy [`0000-template.md`](0000-template.md) to `text/0000-my-feature.md`,
```

Comment:
Where does one find `0000-template.md`? This link does not work.

Reply:
It is not there yet. The nature of this RFC is to propose the content of this file. The template starts [here](https://github.com/lf-lang/rfcs/blob/rfc-process/text/0002-rfc-process.md#abstract-1) in the rendered file